### PR TITLE
Upgrade to python3 in dev-env for ubuntu

### PR DESCRIPTION
**Pull request by kashifest**
Upgrade python in dev-env from python2 to python3 for ubuntu

Solves Issue# 88

### CODE REVIEW

File: ubuntu_install_requirements.sh [hunk 1]
```diff
@@ -18,7 +18,6 @@ sudo apt -y update
 # Install required packages
 
 sudo apt -y install \
-  python-pip \
```
Comment (hardys) [inline ubuntu_install_requirements.sh]: Where does pip3 get installed now this is removed?

Comment (kashifest) [reply ubuntu_install_requirements.sh]: I checked that pyenv is installing the pip3 so we don't need it

**Pull request merged**